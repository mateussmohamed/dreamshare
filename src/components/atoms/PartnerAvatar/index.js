import styled from 'styled-components'

const PartnerAvatar = styled.div`
  width: 120px;
  height: 120px;
  border-radius: 120px;
  background-color: grey;
  background-image: url(${props => props.src});
  background-repeat: no-repeat;
  background-position: center center;
`

export default PartnerAvatar
