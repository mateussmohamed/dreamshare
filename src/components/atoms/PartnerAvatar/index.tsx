import styled from 'styled-components'

interface PartnerAvatarProps {
  src: string;
}

export const PartnerAvatar =
  styled.div <
  PartnerAvatarProps >
  `
  width: 120px;
  height: 120px;
  border-radius: 120px;
  background-color: grey;
  background-image: url(${props => props.src});
  background-repeat: no-repeat;
  background-position: center center;
`
