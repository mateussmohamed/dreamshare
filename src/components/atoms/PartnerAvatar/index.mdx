---
name: PartnerAvatar
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import PartnerAvatar from './index'

# PartnerAvatar
<PropsTable of={PartnerAvatar} />

<Playground>
  <PartnerAvatar />
</Playground>
