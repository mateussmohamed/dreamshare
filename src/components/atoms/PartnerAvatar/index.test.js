import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import PartnerAvatar from './index'

describe('[<PartnerAvatar />]', () => {
  const wrap = (props = {}) => shallow(<PartnerAvatar {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap({})

    expect(wrapper).toMatchSnapshot()
  })
})
