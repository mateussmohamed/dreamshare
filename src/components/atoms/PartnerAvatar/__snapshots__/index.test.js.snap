// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<PartnerAvatar />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "PartnerAvatar-sc-1iftvcv-0",
        "isStatic": false,
        "rules": Array [
          "width:120px;height:120px;border-radius:120px;background-color:grey;background-image:url(",
          [Function],
          ");background-repeat:no-repeat;background-position:center center;",
        ],
      },
      "displayName": "PartnerAvatar",
      "render": [Function],
      "styledComponentId": "PartnerAvatar-sc-1iftvcv-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
